src/main.c

```diff
@@ -125,6 +131,7 @@ static struct nla_policy net_dm_policy[NET_DM_ATTR_MAX + 1] = {
 	[NET_DM_ATTR_HW_ENTRIES]		= { .type = NLA_NESTED },
 	[NET_DM_ATTR_HW_ENTRY]			= { .type = NLA_NESTED },
 	[NET_DM_ATTR_HW_TRAP_COUNT]		= { .type = NLA_U32 },
+	[NET_DM_ATTR_HW_NS]		= { .type = NLA_U32 },
```

Comment:
sorry, just saw that my comments were pending here.  What is a HW Nmespace, and why does it need to be specified separately from the Namespace above?

Reply:
I will reply, though I'm no longer involved in this code, it's not about HW_NS, NS here is basically net namespace.

Reply:
Then why separate it out?

I'm sorry, this doesn't make much sense to me.  Please reopen if you have time to look at this again